api/src/main/resources/org/openmrs/module/isanteplusreports/sql/healthQualReports/pediatricReceivedPcrTest.sql

```diff
@@ -42,16 +42,36 @@ WHERE
         LEFT JOIN isanteplus.patient_prescription pp
             ON phv.patient_id = pp.patient_id
         WHERE
-            DATE(phv.visit_date) BETWEEN :startDate AND :endDate
-            OR (
-                DATE(pp.visit_date) BETWEEN :startDate AND :endDate
-                AND pp.rx_or_prophy = 138405
+            (
+	            DATE(phv.visit_date) BETWEEN :startDate AND :endDate AND phv.encounter_type IN (9,10) -- Paeds initial and followup encounter types
```

Comment:
What is the benefit of converting that field into date when it is date already?

Reply:
Actually, none. I will change this.